src/main/java/frc/robot/Constants.java

```diff
@@ -3,8 +3,34 @@
 
 public final class Constants {
 
+    public static class Swerve{
+        public static final double Rx = 0;
+        public static final double Ry = 0;
+        public static final int [] DRIVE_MOTOR = {0, 0, 0, 0};
+        public static final int [] ANGLE_MOTOR = {0, 0, 0, 0};
+        public static final int VELOCITY_TICKS_PER_UNIT = 0;
+        public static final int DEGREE_TICKS_PER_UNIT = 0;
+        public static final int [] ANGLE_MOTOR_PID_P = {0, 0, 0, 0,};
+        public static final int [] DRIVE_MOTOR_PID_P = {0, 0, 0, 0,};
+        public static final int [] ANGLE_MOTOR_PID_I = {0, 0, 0, 0,};
+        public static final int [] DRIVE_MOTOR_PID_I = {0, 0, 0, 0,};
+        public static final int [] ANGLE_MOTOR_PID_D = {0, 0, 0, 0,};
+        public static final int [] DRIVE_MOTOR_PID_D = {0, 0, 0, 0,};
```

Comment:
The commas at the ends are redundant:
```suggestion
        public static final int [] ANGLE_MOTOR_PID_P = {0, 0, 0, 0};
        public static final int [] DRIVE_MOTOR_PID_P = {0, 0, 0, 0};
        public static final int [] ANGLE_MOTOR_PID_I = {0, 0, 0, 0};
        public static final int [] DRIVE_MOTOR_PID_I = {0, 0, 0, 0};
        public static final int [] ANGLE_MOTOR_PID_D = {0, 0, 0, 0};
        public static final int [] DRIVE_MOTOR_PID_D = {0, 0, 0, 0};
```

Reply:
Also there is no need for the PID in the names